src/Uno.UI/UI/Xaml/UIElement.macOS.cs

```diff
@@ -309,5 +309,140 @@ private void UnregisterTapped(TappedEventHandler handler)
 #endregion
 
 		internal void RaiseTapped(TappedRoutedEventArgs args) => LogUnRegisterPointerReleasedNotImplemented();
+
+		public override void MouseDown(NSEvent evt)
```

Comment:
I take a bet this is mostly copy-pasted from iOS.cs - is there any possibility for greater code reuse?

Reply:
It will need to be refactored, yes. It was duplicated originally because there were too many differences. We'll revisit that once we implement more macOS features.